webapp/courses/views.py

```diff
@@ -459,11 +460,32 @@ def content(request, course_slug, instance_slug, content_slug, **kwargs):
     term_context = context.copy()
     term_context['tooltip'] = True
     terms = Term.objects.filter(instance__course=course).exclude(Q(description__isnull=True) | Q(description__exact='')).order_by('name')
-    terms = [{'name' : term.name, 'slug': slugify(term.name, allow_unicode=True), 
-              'description' : "".join(markupparser.MarkupParser.parse(term.description, request, term_context)).strip(),
-              'span_id' : slugify(term.name, allow_unicode=True) + '-termbank-span'} 
-             for term in terms]
+    term_div_data = []
+    termbank_contents = OrderedDict()
```

Comment:
No need for OrderedDict. Normal dict with sorted should be enough (or normal dict and .sort() after conversion to list).